fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -890,25 +889,16 @@ default boolean recordExists(@Nonnull final Tuple primaryKey, @Nonnull final Iso
 
     /**
      * Get the primary key portion of an index entry.
+     * This API is deprecated. Should use {@link Index#getEntryPrimaryKey(Tuple)} instead.
      * @param index the index associated with this entry
      * @param entry the index entry
      * @return the primary key extracted from the entry
```

Comment:
I think this needs the `@deprecated` Javadoc annotation to make one of the static analysis tools happy.